rust/src/composepost.rs

```diff
@@ -356,25 +356,23 @@ fn compose_postprocess_units(rootfs_dfd: &Dir, treefile: &mut Treefile) -> Resul
 }
 
 #[context("Handling treefile 'default-target'")]
-fn compose_postprocess_default_target(rootfs_dfd: &openat::Dir, target: &str) -> Result<()> {
+fn compose_postprocess_default_target(rootfs: &Dir, target: &str) -> Result<()> {
     /* This used to be in /etc, but doing it in /usr makes more sense, as it's
      * part of the OS defaults. This was changed in particular to work with
      * ConditionFirstBoot= which runs `systemctl preset-all`:
      * https://github.com/projectatomic/rpm-ostree/pull/1425
      */
     let default_target_path = "usr/lib/systemd/system/default.target";
-    rootfs_dfd.remove_file_optional(default_target_path)?;
-    let dest = format!("/usr/lib/systemd/system/{target}");
-    rootfs_dfd.symlink(default_target_path, dest)?;
-
+    rootfs.remove_file_optional(default_target_path)?;
+    rootfs.symlink(target, default_target_path)?;
```

Comment:
Did you change to a relative symlink on purpose here?

Reply:
Yeah, doing absolute symlinks with cap-std requires dropping down a level, and relative symlinks are nicer anyways.